src/pages/about_us.html

```diff
@@ -0,0 +1,228 @@
+    <img class="logo" src="../assets/icons/Finsweet black logo.svg" alt="FinSweet black logo">
+    <nav class="header__nav-links">
+      <a href="../index.html" class="link">Home</a>
+      <a href="#" class="link">Product</a>
+      <a href="#" class="link">Pricing</a>
+      <a href="#" class="link">About us</a>
+      <a href="#" class="link">Blog</a>
+      <a href="#" class="link">Contact</a>
+      <button type="button" class="button button--primary">Free trial</button>
+    </nav>
+  </header>
+
+  <section class="pt-24 flex flex-col gap-16">
+    <div class="section-text section-text--centered">
+      <h1 class="section-text__heading section-text__heading--big">Read our latest blogs
+      </h1>
+      <p class="description description--big">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
+        eirmod tempor
+        invidunt ut labore et
+        dolore
+        magna aliquyam erat.</p>
+    </div>
+    <div class="flex gap-6 justify-center">
+      <img src="../assets/images/person-using-macbook-pro-3861964.png" alt="person using macbook pro">
+      <img src="../assets/images/person-using-macbook-pro-3861964.png" alt="person using macbook pro">
+      <img src="../assets/images/person-using-macbook-pro-3861964.png" alt="person using macbook pro">
+    </div>
+  </section>
+  <section class="pt-[96px]">
+    <div class="section-text section-text--centered">
+      <h4 class="section-text__label">
+        High-quality</h4>
+      <h2 class="section-text__heading max-w-[768px]">We’re a team of data analysts</h2>
+    </div>
+  </section>
+
+  <section class="halved-section halved-section--pushed-down gap-16">
+    <div class="section-text">
+      <h4 class="section-text__label">Our Goals</h4>
+      <h2 class="section-text__heading max-w-[607px]">To upscale your business
+        to the next level</h2>
+      <p class="description desciption--why-us-section ">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
+        diam nonumy eirmod tempor invidunt ut labore et dolore
+        magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.</p>
+    </div>
+    <div class="section-text">
+      <h4 class="section-text__label">Our Vision</h4>
+      <h2 class="section-text__heading max-w-[607px]">To provide solutions for growing companies</h2>
+      <p class="description desciption--why-us-section">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
+        diam nonumy eirmod tempor invidunt ut labore et dolore
+        magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.</p>
+    </div>
+  </section>
+
+  <section class="flex flex-col gap-12 mt-32 px-20">
+    <div class="section-text section-text--centered">
+      <h2 class="section-text__heading">Our corporate values</h2>
+      <p class="description desciption--why-us-section">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
```

Comment:
`desciption--why-us-section` is included without it's default Element (desciption).